scripts/tv-verify-all.sh

```
@@ -0,0 +1,243 @@
+#!/usr/bin/env bash
+# TradingView Indicators verification playbook runner
+set -uo pipefail
+cd "$(dirname "$0")/.."
+ROOT=$(pwd)
+TODAY=$(date +%F)
```

Shellcheck (0.11.0)

[warning] TODAY appears unused. Verify use (or export if used externally).

(SC2034)